### ISSUE TITLE
Python client syntax error in v1.9.2 - duplicate parameter name

### ISSUE THREAD

**Issue by rossjones** [ory/sdk#56]
It appears in v1.9.2 there is a duplicate parameter name in the constructor of PluginConfig which causes
Python to complain of a syntax error.

https://github.com/ory/sdk/blob/master/clients/hydra/python/ory_hydra_client/model/plugin_config.py#L150
```
>   from ory_hydra_client.model.plugin_config import PluginConfig
E     File "/home/circleci/project/.nox/test-3-8-authorization/lib/python3.8/site-packages/ory_hydra_client/model/plugin_config.py", line 150
E       def __init__(self, args, description, documentation, entrypoint, env, interface, ipc_host, linux, mounts, network, pid_host, propagated_mount, work_dir, *args, **kwargs):  # noqa: E501
E       ^
E   SyntaxError: duplicate argument 'args' in function definition
```

This is probably an issue for https://openapi-generator.tech but I guess they won't be able to fix this specific case.

**Comment by aeneasr** [comment]
I think we use the flask generator ( https://openapi-generator.tech/docs/generators/python-flask/ ) which has an option `ensureUniqueParams` which should probably be set to true. Would you be open to give this a shot?

**Comment by rossjones** [comment]
I got the code generation working, but it looks like the generator used is actually 
https://openapi-generator.tech/docs/generators/python rather than the python-flask generator. 
The python-flask generator appears to only be for generating a server.  

The [Python generator](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java) unfortunately does not have that option, and it is not in the global options.  

I'll report an issue over there, but I've no idea how responsive the project is and my Java's a 
couple decades out of date.

**Comment by aeneasr** [comment]
Oh, my bad, you're right, it's the one you linked. Actually this looks like a generator issue as this code shouldn't be touched by the OpenAPI definitions themselves. Sometimes it's also worth checking out if the problem has already been resolved on one of the branches (v5, v5.1, master) in openapi-generator. It's generally possible to use the latest / nightly builds. If it isn't raising an issue there is the way to go. Responsiveness depends on the technical committee - for Golang it's unresponsive, not sure how it is for other languages.

**Comment by aeneasr** [comment]
I'm closing this because it's an upstream issue. If there are any other suggestions on how to fix this please let me know
